# Awesome ATAC-seq analysis

A collection of tools for analysis of chromatin accessibility data.

## QC

- [ATACseqQC](https://bioconductor.org/packages/release/bioc/html/ATACseqQC.html)
- [ATAqC](https://github.com/kundajelab/ataqc)
- [ataqv](https://github.com/ParkerLab/ataqv)

## Single-cell

- [Accessibility Pattern based Epigenomic Clustering (APEC)](https://github.com/QuKunLab/APEC)
- [Bead-based scATAC-seq data Processing (BAP)](https://github.com/caleblareau/bap)
- [Brockman Representation Of Chromatin by K-mers in Mark-Associated Nucleotides.](https://carldeboer.github.io/brockman.html)
- [Cell Ranger ATAC](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac)
- [chromVAR](https://github.com/GreenleafLab/chromVAR)
- [Cicero](https://cole-trapnell-lab.github.io/cicero-release/)
- [cisTopic](https://github.com/aertslab/cisTopic)
- [scABC - single cell Accessibility Based Clustering](https://github.com/SUwonglab/scABC)
- [scAI - single cell Aggregation and Integration method for analyzing single cell multi-omics data](https://github.com/amsszlh/scAI)
- [SCALE](https://github.com/jsxlei/SCALE)
- [SCATE](https://github.com/zji90/SCATE)
- [Scasat (single-cell ATAC-seq analysis tool)](https://github.com/ManchesterBioinference/Scasat)
- [scATAC-pro](https://www.biorxiv.org/content/10.1101/824326v1.abstract)
- [scOpen](https://www.biorxiv.org/content/10.1101/865931v1.abstract)
- [scrat](https://zhiji.shinyapps.io/scrat/)
- [SnapATAC (Single Nucleus Analysis Pipeline for ATAC-seq)](https://github.com/r3fang/SnapATAC)

## Raw data processing pipelines

- [AIAP: a Quality Control and Integrative Analysis Package](https://github.com/Zhang-lab/ATAC-seq_QC_analysis)
- [ATAC-pipe](https://academic.oup.com/bib/article/20/5/1934/5047123)
- [ATAC2GRN](https://github.com/ChioriniLab/ATAC2GRN)
- [ATACProc: a pipeline for processing ATAC-seq data](https://github.com/ay-lab/ATACProc)
- [Basepair](https://www.basepairtech.com/analysis/atac-seq/) 
- [A bioinformatic analysis pipeline for bulk ATAC-seq](https://github.com/shengyongniu/bulk_ATAC_seq)
- [CIPHER](https://github.com/c-guzman/cipher-workflow-platform)
- [ENCODE ATAC-seq pipeline](https://github.com/ENCODE-DCC/atac-seq-pipeline)
- [esATAC](https://www.bioconductor.org/packages/release/bioc/html/esATAC.html)
- [GUAVA: A Graphical User Interface for the Analysis and Visualization of ATAC-seq Data](https://github.com/MayurDivate/GUAVA)
- [I-ATAC](https://www.jax.org/research-and-faculty/resources/i-atac#) - interactive pipeline for the management and pre-processing of ATAC-seq samples.
- [nfcore/atacseq](https://github.com/nf-core/atacseq)
- [PEPATAC](http://pepatac.databio.org/en/latest/)
- [pyflow-atacseq](https://github.com/crazyhottommy/pyflow-ATACseq)
- [snakePipes ATAC-seq](https://snakepipes.readthedocs.io/en/latest/content/workflows/ATAC-seq.html)

## Peak callers

- [F-Seq](http://fureylab.web.unc.edu/software/fseq/)
- [Genrich](https://github.com/jsh58/Genrich)
- [HMMRATAC](https://github.com/LiuLabUB/HMMRATAC)
- [HOMER](http://homer.ucsd.edu/homer/)
- [Hotspot2](https://github.com/Altius/hotspot2)
- [MACS](https://github.com/taoliu/MACS)
- [PeaKDEck](https://www.ccmp.ox.ac.uk/peakdeck)

## Motif enrichment/TF footprinting/TFBS

- [BiFET](https://github.com/UcarLab/BiFET)
- [BinDNase](https://research.cs.aalto.fi/csb/software/index.shtml)
- [CENTIPEDE](http://centipede.uchicago.edu/)
- [DeFCoM](https://bitbucket.org/bryancquach/defcom/src)
- [DNase2TF](https://sourceforge.net/projects/dnase2tfr/)
- [HINT-ATAC](https://link.springer.com/article/10.1186/s13059-019-1642-2)
- [HOMER](http://homer.ucsd.edu/homer/)
- [MEME Suite](http://meme-suite.org/)
- [MMARGE](https://github.com/vlink/marge/)
- [PIQ](http://piq.csail.mit.edu/)
- [Romulus](https://github.com/ajank/Romulus)
- [TEPIC](https://github.com/schulzlab/TEPIC)
- [TOBIAS](https://github.com/loosolab/TOBIAS)
- [TRACE](https://www.biorxiv.org/content/10.1101/801001v1.abstract)
- [Wellington](https://github.com/jpiper/pyDNase)

## Differential accessibility

- [ALTRE](https://mathelab.github.io/ALTRE/)
- [DAStk](https://github.com/Dowell-Lab/DAStk)
- [diffTF](https://difftf.readthedocs.io/en/latest/)

## Nucleosome positioning

- [HMMRATAC](https://academic.oup.com/nar/article/47/16/e91/5519166)
- [NucleoATAC](https://nucleoatac.readthedocs.io/en/latest/)
- [NucTools](https://link.springer.com/article/10.1186/s12864-017-3580-2)

## Region enrichment

- [Annotatr](https://bioconductor.org/packages/release/bioc/html/annotatr.html)
- [GIGGLE](https://github.com/ryanlayer/giggle)
- [GREAT](http://great.stanford.edu/public/html/)
- [LOLA](https://github.com/nsheff/LOLA)
- [regioneR](http://bioconductor.org/packages/release/bioc/html/regioneR.html)

## Uncategorized

- [AtacWorks](https://github.com/clara-genomics/AtacWorks): correct noisy/low-quality ATAC-seq data to obtain improved peak calls
- [BeCorrect](https://github.com/Zhang-lab/BeCorrect): correct the batch effect of ATAC-seq signal density for visualization purposes
- [seqOutBias](https://guertinlab.github.io/seqOutBias/): correction of enzymatic sequence bias
- [sequence bias correction](https://github.com/txje/sequence-bias-adjustment)
