# ATAC-seq tools

## QC

- [ATAqC](https://github.com/kundajelab/ataqc)
- [ATACseqQC](https://bioconductor.org/packages/release/bioc/html/ATACseqQC.html)


## Single-cell
- [scATAC-pro](https://www.biorxiv.org/content/10.1101/824326v1.abstract)
- [scOpen](https://www.biorxiv.org/content/10.1101/865931v1.abstract)
- [SnapATAC (Single Nucleus Analysis Pipeline for ATAC-seq)](https://github.com/r3fang/SnapATAC)
- [Bead-based scATAC-seq data Processing (BAP)](https://github.com/caleblareau/bap)
- [Cell Ranger ATAC](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac)
- [Scasat (single-cell ATAC-seq analysis tool)](https://github.com/ManchesterBioinference/Scasat)


## Standalone (or mini-pipeline) Downstream Analysis Tools

- [diffTF](https://difftf.readthedocs.io/en/latest/)
- [chromVAR](https://github.com/GreenleafLab/chromVAR)
- [single cell Accessibility Based Clustering (scABC)](https://github.com/SUwonglab/scABC)
- [Cicero](https://cole-trapnell-lab.github.io/cicero-release/)
- [Brockman Representation Of Chromatin by K-mers in Mark-Associated Nucleotides.](https://carldeboer.github.io/brockman.html)
- [cisTopic](https://github.com/aertslab/cisTopic) 
- [scrat](https://zhiji.shinyapps.io/scrat/)

## Raw data processing pipelines

- [PEPATAC](http://pepatac.databio.org/en/latest/)
- [ATAC-pipe](https://academic.oup.com/bib/article/20/5/1934/5047123)
- [pyflow-atacseq](https://github.com/crazyhottommy/pyflow-ATACseq)
- [AIAP: a Quality Control and Integrative Analysis Package](https://github.com/Zhang-lab/ATACseq_benchmarking)
- [ATACProc: a pipeline for processing ATAC-seq data](https://github.com/ay-lab/ATACProc)
- [esATAC](https://www.bioconductor.org/packages/release/bioc/html/esATAC.html)
- [snakePipes ATAC-seq](https://snakepipes.readthedocs.io/en/latest/content/workflows/ATAC-seq.html)
- [I-ATAC](https://www.jax.org/research-and-faculty/resources/i-atac#) - interactive pipeline for the management and pre-processing of ATAC-seq samples.
- [ENCODE ATAC-seq pipeline](https://github.com/ENCODE-DCC/atac-seq-pipeline)
- [nfcore/atacseq](https://github.com/nf-core/atacseq)
- [Basepair](https://www.basepairtech.com/analysis/atac-seq/) 
- [GUAVA: A Graphical User Interface for the Analysis and Visualization of ATAC-seq Data](https://github.com/MayurDivate/GUAVA)
- [CIPHER](https://github.com/c-guzman/cipher-workflow-platform)


## Peak callers

- [HINT-ATAC](https://link.springer.com/article/10.1186/s13059-019-1642-2)

## Footprinting

- [TRACE](https://www.biorxiv.org/content/10.1101/801001v1.abstract)
- CENTIPEDE
- seqoutbias

## Nucleosome positioning

- [NucleoATAC](https://nucleoatac.readthedocs.io/en/latest/)
- [hmmratac](https://academic.oup.com/nar/article/47/16/e91/5519166)
- [NucTools](https://link.springer.com/article/10.1186/s12864-017-3580-2)


## Uncategorized
